### ISSUE TITLE
System Upgrade window too big

### ISSUE THREAD

**Issue by EchoPouet** [linuxmint/mintupdate#733]
Hello,

The system upgrade window is too big in my little screen. Also I can't resize it.

Screenshot to show the problem with buttons:
![MintUpgradeTooBig](https://user-images.githubusercontent.com/5129154/167300810-9af53193-0c8f-4a76-bbf6-66bf2aa0192e.png)

I haven't problem with others windows.

**Comment by kneekoo** [comment]
Yeah, that window is too big for no real reason. It can easily be smaller and it will be just as useful.
